XYZDrivers/src/main/java/com/xyzdrivers/services/InsertPaymentService.java

```diff
@@ -44,10 +47,34 @@ public void InsertPayment(MembershipPayment mp) throws IllegalAccessException, S
         ps.setDate(4, Date.valueOf(mp.getDate()));
         ps.setTime(5, Time.valueOf(mp.getTime()));
 
-        ps.executeUpdate();
+                ps.executeUpdate();
+        
+        String query2 = "SELECT BALANCE FROM MEMBERS WHERE ID = ?";
+        
+        PreparedStatement ps2 = con.prepareStatement(query2);
+        
+        ps2.setString(1, mp.getMemberID());
+        
+        ResultSet rs = ps2.executeQuery();
+        
+        while(rs.next()){
```

Comment:
Please use the sql service for this. We need to keep the data access in a single layer to isolate responsibilities.